ssn/ssn_separated/ssn.ttl

```diff
@@ -670,9 +658,9 @@ ssn:Observation rdf:type owl:Class ;
                                   owl:minCardinality "0"^^xsd:nonNegativeInteger
                                 ] ,
                                 [ rdf:type owl:Restriction ;
-                                  owl:onProperty ssn:featureOfInterest ;
+                                  owl:onProperty sosa:featureOfInterest ;
```

Comment:
Indeed, there was no connection between FeatureOfInterest and ObservableProperty in SOSA because that would have involved establishing something akin to the ISO 19109 General Feature Model, which is way beyond the scope of SOSA. It would also have likely involved either rdfs semantics (domain and range) or OWL semantics (owl:Restriction) which we agreed would not be used in SOSA.